Error building 2.0.1 with Android on API 35

### Before submitting a new issue

- [x] I tested using the latest version of the library, as the bug might be already fixed.
- [x] I tested using a [supported version](https://github.com/reactwg/react-native-releases/blob/main/docs/support.md) of react native.
- [x] I checked for possible duplicate issues, with possible answers.

### Bug summary

#  Android build failure with TrueSheetPackage and react-native-date-picker in Expo managed workflow

## Description
When building my Expo SDK52) React Native (0.76.7 new arch) app for Android, I'm encountering a persistent error that prevent successful builds:

1. Cannot find `TrueSheetPackage` from `@lodev09/react-native-true-sheet`:error: cannot find symbol import com.truesheet.TrueSheetPackage; symbol: class TrueSheetPackagelocation: package com.truesheet

Is there something I need to put in my app.json related to this package or an expo config plugin?

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

/Users/sam/Documents/GitHub/sam/project/apps/mobile/android/app/build/generated/autolinking/src/main/java/com/facebook/react/PackageList.java:16: error: cannot find symbol
import com.truesheet.TrueSheetPackage;
                   ^
 symbol:   class TrueSheetPackage
 location: package com.truesheet
 
/Users/sam/Documents/GitHub/sam/project/apps/mobile/android/app/build/generated/autolinking/src/main/java/com/facebook/react/PackageList.java:135: error: cannot find symbol
     new TrueSheetPackage(),
         ^
 symbol:   class TrueSheetPackage
 location: class PackageList
2 errors
```

### Library version

2.0.1

### Environment info

React native 0.76.7 
New arch
Expo SDK 52
Android build

### Steps to reproduce

1. Run a build with latest

ı had the same problem when ı tried to build my application .                                                                                                          error: cannot find symbol
import com.truesheet.TrueSheetPackage;
                    ^
  symbol:   class TrueSheetPackage
  location: package com.truesheet
/Volumes/SSD/XNote-Mobile-Expo/android/app/build/generated/autolinking/src/main/java/com/facebook/react/PackageList.java:109: error: cannot find symbol
      new TrueSheetPackage(),
          ^
  symbol:   class TrueSheetPackage
  location: class PackageList
2 errors

FAILURE: Build failed with an exception.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -45,7 +45,7 @@ def supportsNamespace() {
 
 android {
   if (supportsNamespace()) {
-    namespace "com.truesheet"
+    namespace "com.lodev09.truesheet"
 
     sourceSets {
       main {

